dispatch/static/manager/src/js/index.js

```diff
@@ -26,7 +28,14 @@ const store = createStore(
     loadingBar: loadingBarReducer,
     modal: modalReducer
   }),
-  applyMiddleware(thunk, router, promiseMiddleware(), loadingBarMiddleware())
+  applyMiddleware(
+    thunk,
+    router,
+    promiseMiddleware(),
+    loadingBarMiddleware({
+      promiseTypeSuffixes: ['PENDING', 'FULFILLED', 'REJECTED'],
```

Comment:
Oh, I see.

Yeah, that's probably something we should standardize 😛 . Let's issue it up